File: src/main/Java/boj_step7/boj2738.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2738 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int n = Integer.parseInt(tokenizer.nextToken());
        int m = Integer.parseInt(tokenizer.nextToken());
        int[][] procession = new int[n][m];
        for (int i = 0; i < n; i++) {
            tokenizer = new StringTokenizer(reader.readLine());
            for (int j = 0; j < m; j++) procession[i][j] = Integer.parseInt(tokenizer.nextToken());
        }
        for (int i = 0; i < n; i++) {
            tokenizer = new StringTokenizer(reader.readLine());
            for (int j = 0; j < m; j++) {
                procession[i][j] += Integer.parseInt(tokenizer.nextToken());
                writer.append(procession[i][j] + " ");
            }
            writer.newLine();
        }
        writer.close();
    }
}

File: src/main/kotlin/others/boj2738.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (n, m) = r.readLine()!!.split(" ").map { it.toInt() }
    var a = Array(n) { IntArray(m) }
    val b = Array(n) { IntArray(m) }
    a.forEach {
        val arr = r.readLine()!!.split(" ").map { it.toInt() }
        for (i in 0 until m) it[i] = arr[i]
    }
    b.forEach {
        val arr = r.readLine()!!.split(" ").map { it.toInt() }
        for (i in 0 until m) it[i] = arr[i]
    }
    for(i in 0 until a.size) {
        for (j in 0 until a[i].size) {
            a[i][j] += + b[i][j]
            w.write(a[i][j].toString() + " ")
        }
        w.write("\n")
    }
    w.close()
}